funfact/model/_factorization.py

```diff
@@ -44,15 +45,30 @@ def __init__(self, tsrex, initialize=True, nvec=1):
             tsrex = tsrex | self._leaf_initializer
         tsrex = tsrex | self._shape_analyzer | self._einspec_generator
         self._tsrex = tsrex
+        self._nvec = nvec
 
     @property
     def tsrex(self):
         return self._tsrex
 
+    @property
+    def nvec(self):
+        return self._nvec
+
     @property
     def shape(self):
         return self._tsrex.root.shape
 
+    @classmethod
+    def devectorize(cls, factorization, slice: int):
```

Comment:
I feel that `i` or `instance` might be a better name than `slice`, since it is a single integer.

Reply:
Renamed to `instance` in latest commit.